src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -61,29 +94,31 @@ export const UserOrcid = ({ user }: UserOrcidProps) => {
           method: 'PATCH',
           data: { orcid },
         });
-        if (isSuccessStatus(addOrcidResponse.status)) {
-          dispatch(setNotification({ message: t('feedback.success.update_orcid'), variant: 'success' }));
-          cristinPersonQuery.refetch();
-        } else if (isErrorStatus(addOrcidResponse.status)) {
-          dispatch(setNotification({ message: t('feedback.error.update_orcid'), variant: 'success' }));
+        const orcidCredentials = getOrcidCredentials(history.location.search, orcidInfoResponse.data.id);
+        if (!orcidCredentials) {
+          dispatch(setNotification({ message: t('feedback.error.storing_orcid_credentials'), variant: 'warning' }));
```

Comment:
dette er kanskje også en error?
```suggestion
          dispatch(setNotification({ message: t('feedback.error.storing_orcid_credentials'), variant: 'error' }));
```